modules/Hoot/ui/modals/modifyFolder.js

```diff
@@ -88,7 +88,7 @@ export default class ModifyFolder {
             str              = node.value,
 
             reservedWords    = [ 'root', 'dataset', 'dataset', 'folder' ],
-            unallowedPattern = new RegExp( /[~`#$%\^&*+=\-\[\]\\';\./!,/{}|\\":<>\?|]/g ),
+            unallowedPattern = new RegExp( /[~`#$%\^&*+=\[\]\\';/!,/{}|\\":<>\?|]/g ),
```

Comment:
I removed the `-` and `.` from this since the grail pull directory default name includes both of these characters